admin/controller/payment/razorpay.php

```diff
@@ -311,6 +311,466 @@ public function uninstall(): void {
 		}
 	}
 
+    protected function getForm()
+    {
+        $data['text_form'] = !isset($this->request->get['entity_id']) ? $this->language->get('text_add') : $this->language->get('text_edit');
+
+        if (isset($this->request->get['entity_id']))
+        {
+            $data['entity_id'] = (int)$this->request->get['entity_id'];
+        }
+        else
+        {
+            $data['entity_id'] = 0;
+        }
+
+        if (isset($this->error['warning']))
+        {
+            $data['error_warning'] = $this->error['warning'];
+        }
+        else
+        {
+            $data['error_warning'] = '';
```

Comment:
@abdulwahidsharief , assign all empty variables at the start. no need to use else block.